### DISCUSSION TITLE
这个库官方还维护吗？

### DISCUSSION THREAD

**Discussion by hao-kuai** [ElemeFE/element#22545] [Q&A] [answered]
官方还负责维护吗？有bug还负责解决吗？

**Comment by webvs2** [comment]
Hello, do you have any questions?

**Reply by hao-kuai** [reply]
Select 组件遇到问题 [#21290](https://github.com/ElemeFE/element/issues/21290),这个问题提出来很久了，也有人提交的解决方法，一直没有被解决，这个问题是一个bug吗？如果是bug，还有修复计划啊？

**Reply by webvs2** [reply, marked as answer]
Although the update frequency has been adjusted, don't worry, it is in the update plan

**Reply by hao-kuai** [reply]
期待新版本！大概什么时候发布新版本？迫不及待地学习官方解决方案。
